server/centralized.go

```diff
@@ -226,6 +236,16 @@ func (c *Centralized) batchReads() {
 				c.log.Trace.Printf("Read: add to batch, size=%v\n", len(c.ReadBatch))
 			}
 			continue
+		case <-tick:
+			if len(c.ReadBatch) > 0 {
+				for len(c.ReadBatch) < config.ReadBatch {
```

Comment:
Is it necessary to pad empty requests? Why can't we just issue a batchRead with whatever we have?